tests/lit.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python
```

Comment:
lit is not included in macport's LLVM install. And I wasn't sure about where you want to get it from (PyPi, developer install, LLVM source...). My preference is also to not have this in SDC's repo.